components/map/MapBackground.tsx

```diff
@@ -32,6 +111,66 @@ const styles = StyleSheet.create({
     width: '100%',
     height: '125%',
   },
-});
+}); 
+//renvoi les informations des moyens lié à l'intervention en cours
+async function MeansLoading() {
+  const {data , error}= await supabase.from(Tables.INTERVENTIONS_MEANS_LINK).select('mean_id,'+Tables.MEANS+'(id)').eq(Columns.INTERVENTION_ID,usePathname().split('/')[2])
+  
+  if (error) {
+    console.log("1")
+    console.log (error)
+  } else {
+    // TO DO Problème pour récupérer la localisation ,conversion en location ? 
+    console.log("2")
+    console.log(data)
+    let data2 = data as unknown  as Mean 
+    
+    let data3 = data2.location as unknown as location
+    console.log(data3) 
+  }
+}
+//renvoi les informations lié à l'intervention en cours
```

Comment:
La doc c'est comme en java:
```suggestion
/**
 * Renvoie les informations liées à l'intervention en cours
 */
```